src/test/resources/application.yaml

```diff
@@ -91,7 +93,8 @@ spring:
     mode: TEXT
     suffix: .sql
 
-  jersey.application-path: rest
+  # this must be `camunda` to match module descriptor path patterns
+  jersey.application-path: camunda
```

Comment:
Update [here](https://github.com/folio-org/mod-workflow/pull/117/files#diff-5f1399fae635ac57e696d503eb2135f7575b3a59a486e21613d3343f8f2a3243R50) when refactoring module descriptor to workflow-engine.